internal/compiler/widgets/cosmic/styling.slint

```diff
@@ -69,6 +69,12 @@ export global CosmicPalette {
     out property <brush> state-secondary: dark-color-scheme ? #000000 : #ffffff;
 }
 
+export global CosmicLayoutSettings {
```

Comment:
Any reason why there are both a `<Style>LayoutSettings` and a `LayoutSettings` instead of just having one  `LayoutSettings` being re-exported.

Each global has a cost, and we should avoid having too many of them.